onnxruntime/python/tools/transformers/models/bert/eval_squad.py

```diff
@@ -207,7 +218,7 @@ def main():
             squad_v2_format=True,
         )
 
-        result["provider"] = "CUDAExecutionProvider" if args.use_gpu else "CPUExecutionProvider"
+        result["provider"] = args.provider if use_gpu else "CPUExecutionProvider"
```

Comment:
I am wondering why this cannot be just `result["provider"] = args.provider`

Reply:
changed to what you suggest.